repository/flink/docs/demo/financial-fraud/README.md

```diff
@@ -17,91 +17,103 @@ The architecture follows more or less the [SMACK stack architecture](https://mes
 
 ## Prerequisites
 
-Before you get started:
+Before you get started, make sure you have a cluster at hand with enough resources.
 
-- Make sure you have a cluster at hand with enough resources, e.g.:
+### Run on [KinD](https://github.com/kubernetes-sigs/kind)
 
-
-For [KinD](https://github.com/kubernetes-sigs/kind):
-```
+```bash
 # Use make from operators repo:
-make create-cluster
+$ make create-cluster
 
 # or modify your own KinD cluster:
-kind create cluster
-export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
-kubectl delete storageclass standard
-kubectl apply -f https://raw.githubusercontent.com/rancher/local-path-provisioner/master/deploy/local-path-storage.yaml
-kubectl annotate storageclass --overwrite local-path storageclass.kubernetes.io/is-default-class=true
+$ kind create cluster
+$ export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
+$ kubectl delete storageclass standard
```

Comment:
The latest release of [kind (0.7.0)](https://github.com/kubernetes-sigs/kind/releases/tag/v0.7.0) has all this stuff built in so you don't need these steps any more.  Might be worth mentioning in case folks are tracking the bleeding edge.

Reply:
Do you mind fixing it in a follow-up PR? I'll merge this now so the demo works again.